src/gui/folderman.cpp

```diff
@@ -1610,18 +1610,21 @@ void FolderMan::slotLeaveShare(const QString &localFile, const QByteArray &folde
 }
 
 void FolderMan::trayOverallStatus(const QList<Folder *> &folders,
-    SyncResult::Status *status, bool *unresolvedConflicts)
+                                  SyncResult::Status *status,
+                                  bool *unresolvedConflicts,
+                                  const ProgressInfo **overallProgressInfo)
```

Comment:
same comment about the position of the `const` keyword
```suggestion
                                  ProgressInfo ** const overallProgressInfo)
```